README.md

```diff
@@ -5,9 +5,11 @@ Python container images
 Images available on Quay are:
 * CentOS 7 [python-38](https://quay.io/repository/centos7/python-38-centos7)
 * RHEL 8 [python-39-minimal](https://quay.io/repository/sclorg/python-39-minimal-el8)
+* CentOS Stream 8 [python-39-c8s](https://quay.io/repository/sclorg/python-39-c8s)
 * CentOS Stream 9 [python-39-c9s](https://quay.io/repository/sclorg/python-39-c9s)
-* Fedora [python-39](https://quay.io/repository/fedora/python-39)
```

Comment:
The Fedora version where Python 3.9 is the main one – 35 – is EOL since November 2021. I don't think it's a good idea to promote unmaintained releases. What do you think?